tsconfig-ts.json

```
@@ -6,7 +6,7 @@
     "module": "CommonJS",
     "skipLibCheck": true,
     "strict": true,
-    "target": "ES2019",
+    "target": "ES2021",
```

Since our `nvmrc` stipulates Node 20, we can actually push this to 2022[^tsconfig]

```suggestion
    "target": "es2022",
```

[^tsconfig]: For a new project today, I would actually just extend [**@tsconfig/node20**](https://www.npmjs.com/package/@tsconfig/node20), but it doesn’t really make sense for us to retrofit it to a legacy codebase